utils_cv/detection/plot.py

```diff
@@ -56,7 +56,12 @@ def plot_boxes(
     """
     if len(bboxes) > 0:
         draw = ImageDraw.Draw(im)
+        font = get_font(size=plot_settings.text_size)
+
         for bbox in bboxes:
+            # do not draw background bounding boxes
+            if bbox.label_idx == 0:
```

Comment:
this is why i'm worried about the idea of adding the little box int he corner. If people wanna write their own function for plotting, they'd have to know this weird edge-case.

Reply:
Fully agree. I don't like it either.

Reply:
Just created an issue to keep track of this. But main problem I see is that edits would at the very least be required in the "references" folder which is not good either.
https://github.com/microsoft/ComputerVision/issues/372